jdk7/src/main/java/pcap/jdk7/internal/NativeMappings.java

```diff
@@ -58,6 +61,14 @@ class NativeMappings {
   private static final Map<String, Object> NATIVE_LOAD_LIBRARY_OPTIONS =
       new HashMap<String, Object>();
 
+  static {
+    File NPCAP_DIR = Paths.get(System.getenv("SystemRoot"), "System32", "Npcap").toFile();
```

Comment:
It better to move this line in to if block (Platform.isWindows() && System.getProperty("jna.library.path")'/